src/main/clojure/clara/rules/durability.clj

```diff
@@ -440,45 +435,27 @@
      (eng/assemble {:rulebase rulebase
                     :memory (eng/local-memory rulebase
                                               (clara.rules.engine.LocalTransport.)
-                                              (eng/options->activation-group-sort-fn opts)
-                                              (eng/options->activation-group-fn opts)
+                                              (:activation-group-sort-fn rulebase)
+                                              (:activation-group-fn rulebase)
                                               ;; TODO: Memory doesn't seem to ever need this or use
                                               ;; it.  Can we just remove it from memory?
-                                              get-alphas-fn)
+                                              (:get-alphas-fn rulebase))
```

Comment:
We shouldn't call `opts->get-alphas-fn` on the given `opts` anymore above either.  Relates to
https://github.com/cerner/clara-rules/pull/235/files#r87932559